SwiftCheck/Modifiers.swift

```diff
@@ -71,10 +66,84 @@ public func == <T : protocol<Arbitrary, Equatable>>(lhs : Static<T>, rhs : Stati
 	return lhs.getStatic == rhs.getStatic
 }
 
-public func <^><A : Arbitrary, B : Arbitrary>(f: A -> B, ar : Static<A>) -> Static<B> {
-	return Static<B>(f(ar.getStatic))
+/// Generates an array of arbitrary values of type A.
+public struct ArrayOf<A : Arbitrary> : Arbitrary, Printable {
```

Comment:
Investigate whether ArrayLiteralConvertible allows us to silently instantiate one of these things